quill-core/src/main/scala/io/getquill/Query.scala

```diff
@@ -17,10 +17,10 @@ sealed trait Query[+T] {
 
   def groupBy[R](f: T => R): Query[(R, Query[T])]
 
-  def min[U >: T](implicit n: Numeric[U]): Option[T]
-  def max[U >: T](implicit n: Numeric[U]): Option[T]
+  def min[U >: T](implicit o: Ordering[U]): Option[T]
+  def max[U >: T](implicit o: Ordering[U]): Option[T]
   def avg[U >: T](implicit n: Numeric[U]): Option[T]
-  def sum[U >: T](implicit n: Numeric[U]): T
+  def sum[U >: T](implicit n: Numeric[U]): Option[T]
```

Comment:
Just saw https://github.com/getquill/quill/pull/69. Maybe this change is here by accident?